vector/src/main/AndroidManifest.xml

```diff
@@ -193,10 +193,15 @@
 
                 <data android:scheme="https" />
                 <data android:host="riot.im" />
-                <data android:host="app.element.io" />
-                <data android:host="mobile.element.io" />
-                <data android:host="develop.element.io" />
-                <data android:host="staging.element.io" />
+            </intent-filter>
+            <intent-filter android:autoVerify="true">
+                <action android:name="android.intent.action.VIEW" />
+
+                <category android:name="android.intent.category.DEFAULT" />
+                <category android:name="android.intent.category.BROWSABLE" />
+
+                <data android:scheme="https" />
+                <data android:host="*.element.io" />
```

Comment:
captures all subdomains of `element.io` to avoid needing to host a assetlinks.json on every subdomain